sources/packages/backend/apps/db-migrations/src/sql/EducationProgramsOfferings/Add-aviation-columns.sql

```diff
@@ -0,0 +1,37 @@
+ALTER TABLE
+  sims.education_programs_offerings
+ADD
+  COLUMN is_aviation_offering VARCHAR(50) NOT NULL DEFAULT 'no';
+
+COMMENT ON COLUMN sims.education_programs_offerings.is_aviation_offering IS 'Offering is an aviation offering?';
+
+ALTER TABLE
+  sims.education_programs_offerings
+ADD
+  COLUMN aviation_credential_type VARCHAR(50);
+
+COMMENT ON COLUMN sims.education_programs_offerings.aviation_credential_type IS 'Aviation offering credential type.';
+
+ALTER TABLE
+  sims.education_programs_offerings
+ALTER COLUMN
+  is_aviation_offering DROP DEFAULT;
+
+ALTER TABLE
+  sims.education_programs_offerings_history
+ADD
+  COLUMN is_aviation_offering VARCHAR(50) NOT NULL DEFAULT 'no';
```

Comment:
History table columns are created without relationships or constraints. They can all be nullable.